cmd/manta-init.js

```diff
@@ -802,7 +794,7 @@ var pipelineFuncs = [
          */
         extra.metadata['MAKO_HTTP_KEEPALIVE_TIMEOUT'] = 86400;
 
-        // This is filled in when marlin is deployed.
+        // XXX Only needed until jobs are fully removed everywhere.
```

Comment:
What breaks if SERVER_COMPUTE_ID_MAPPING isn't set? Ah, this is how a mako gets its ID?

Reply:
yeah, it's used at least in mako's setup.sh. So I think if we want to remove this field, we'll want to do that as an update there and here and also a look around to see where else it might still be referenced.